Golem/Golem.cs

```diff
@@ -11,6 +11,11 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 
+#if OS_WINDOWS
```

Comment:
I saw `RuntimeInformation.IsOSPlatform` and https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.conditionalattribute?view=net-8.0 elsewhere in the codebase.
Do I understand correctly that it's impossible to use it for includes?